### DISCUSSION TITLE
\anon报错

### DISCUSSION THREAD

**Discussion by Nono-404** [note286/xduts#110] [Q&A] [answered]
> 查看 #61 中的评论。

![image](https://user-images.githubusercontent.com/56815855/228181710-88c79bdf-10e3-455a-a554-55ffb9c48232.png)
在个人简介里使用\anno{}报错：Undefined control sequence.大佬知道这是什么问题吗

_Originally posted by @Nono-404 in https://github.com/note286/xduts/discussions/94#discussioncomment-5450522_

**Comment by note286** [comment, marked as answer]
你的问题无法复现。

**Reply by Nono-404** [reply]
> 你的问题无法复现。

不好意思，我设置的问题，非常感谢您的回答

**Reply by note286** [reply]
我好奇你设置了啥会导致这个问题。

**Reply by Nono-404** [reply]
> 我好奇你设置了啥会导致这个问题。

哈哈哈是我其中一个anon打成了anno，非常低级的错误，下次好好检查检查一下再问

**Reply by note286** [reply]
好吧。。。。
